app/controllers/api/v1/tokens_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class TokensController < Doorkeeper::TokensController
+      include ErrorHandlerConcern
+
+      def revoke
```

Comment:
Maybe we can add a comment at top of this function

```
   # Overriden from doorkeeper as the doorkeeper revoke action does not return response according to json-api spec
```

Reply:
Also updated comments in the other method overridden ;-)